lib/markdown_generators.js

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env node
+
+const yaml = require("js-yaml");
+const markdown = require("markdown-builder");
```

Comment:
It looks like this library's repo is archived by the owner. Is there an alternative we can use?

Reply:
This is a simple string builder type thing that has convenience functions for generating markdown snippets.  I haven't found anything exactly like it, unfortunately.  I could just use string literals to build the markdown I need, rather than depend on the library but it could get tedious depending on what special casing needs to be done once we get in to the thick of looking at individual courses.